db/migrate/20191215145706_add_edited_at_to_messages.rb

```diff
@@ -0,0 +1,5 @@
+class AddEditedAtToMessages < ActiveRecord::Migration[5.2]
+  def change
+    add_column :messages, :edited_at, :datetime
```

Comment:
@mstruve I like to consider `updated_at` as a reserved Rails attribute because it's tied to all sorts of default behavior.

If we were to _say_ run a script that re-formatted a bunch of old messages without changing the content, we wouldn't want the UI to display `edited_at` which I believe is specific to the user interaction and not the model lifecycle.